roles/sync/templates/pihole_sync.j2

```diff
@@ -35,4 +43,29 @@ if [[ ! $(ip a | grep {{ sync_target }}) ]]; then
       sudo cp --preserve=timestamps $sync_dir/05-pihole-custom-cname.conf $pihole_dir/dnsmasq.d
     fi
   fi
+
+  RSYNC_WILDCARD=$(rsync -a --info=name -e "ssh $key $host_key_check" $target:$pihole_dir/dnsmasq.d/02-cluster.vert-wildcard.conf $sync_dir)
```

Comment:
What is the reason to sync this file separately when you later on sync the whole dnsmasq.d directory?